build.gradle

```diff
@@ -8,6 +8,7 @@ plugins {
   id 'com.github.ben-manes.versions' version '0.38.0'
   id 'org.sonarqube' version '3.1.1'
   id 'uk.gov.hmcts.java' version '0.12.2'
+  id 'org.flywaydb.flyway' version '7.0.0'
```

Comment:
Version mismatch with dependencies:org.flywaydb:flyway-core.

Reply:
Fixed in commit https://github.com/hmcts/cpo-case-payment-orders-api/pull/16/commits/f47a41148bdbc4123ae3b8761b453b6e8b535f8d